pinot-hadoop/src/main/java/com/linkedin/pinot/hadoop/job/mapper/HadoopSegmentCreationMapReduceJob.java

```diff
@@ -280,13 +305,17 @@ private RecordReaderConfig getReaderConfig(FileFormat fileFormat) throws IOExcep
         case CSV:
           if(_readerConfigFile == null) {
             readerConfig = new CSVRecordReaderConfig();
-	  }
-	  else {
-	    LOGGER.info("Reading CSV Record Reader Config from: {}", _readerConfigFile);
-	    readerConfig = new ObjectMapper().readValue(new File(_readerConfigFile), CSVRecordReaderConfig.class);
-	    LOGGER.info("CSV Record Reader Config: {}", readerConfig.toString());
           }
-	  break;
+          else {
+            LOGGER.info("Reading CSV Record Reader Config from: {}", _readerConfigFile);
+            // URL configURL = new URL(_readerConfigFile);
+
+            Path p = new Path(_readerConfigFile);
```

Comment:
is it possible to add a test case for csv generation/reading? What is the issue you ran into? We don't usually generate CSV internally, so I am not too familiar with this part.

Reply:
Not sure how and where to add the test case for this. Earlier the code was only compitable to read the config from the local file system. When the defaultFS of hadoop is other than file, this was breaking.